### PR TITLE
13980_admin_theme

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/13980

### CODE REVIEW

**Comment by hoanganhd-1958** [comment]
ready

File: app/Http/Controllers/LangController.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    private $langActive = [
+        'vi',
+        'en',
+    ];
+    public function changeLang(Request $request, $lang)
+    {
+        if (in_array($lang, $this->langActive)) {
+            $request->session()->put(['lang' => $lang]);
+            return redirect()->back();
```
Comment (BunnyPi04) [inline app/Http/Controllers/LangController.php:18]: thiếu dòng trắng trước return

File: app/Http/Middleware/LangMiddleware.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if ($lang=$request->session()->get('lang')) {
```
Comment (BunnyPi04) [inline app/Http/Middleware/LangMiddleware.php]: convention

File: resources/js/components/StoryComponent.vue [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="container">
```
Comment (BunnyPi04) [inline resources/js/components/StoryComponent.vue]: indent = 4 spaces

File: routes/web.php [hunk 4]
```diff
@@ -10,6 +10,11 @@
 | contains the "web" middleware group. Now create something great!
 |
 */
+
+Route::get('/', function () {
+    return view('layouts.master');
```
Comment (BunnyPi04) [inline routes/web.php]: ko return view trong route

**Comment by ghost** [comment]
ready

File: resources/views/layouts/master.blade.php [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+    <meta name="locale" content="{{ App::getLocale() }}" />
+
+    <title>Stories | Admin</title>
```
Comment (BunnyPi04) [inline resources/views/layouts/master.blade.php]: i18n

File: resources/views/layouts/master.blade.php [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+                    <a class="nav-link" data-toggle="dropdown" href="#" aria-expanded="true">
+                        <i class="fas fa-globe-asia"></i>
+                    </a>
+                    <div class="dropdown-menu dropdown-menu-lg dropdown-menu-right">
+                        <a href="{{ route('lang',['lang' => 'vi']) }}" class="dropdown-item">
+                            <!-- Message Start -->
+                            <div class="media">
+                                <div class="media-body">
+                                    <h3 class="dropdown-item-title">
+                                        Tiếng Việt
```
Comment (sonnguyen8x) [inline resources/views/layouts/master.blade.php]: i18n

File: resources/views/layouts/master.blade.php [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+                                    </h3>
+                                </div>
+                            </div>
+                            <!-- Message End -->
+                        </a>
+                        <div class="dropdown-divider"></div>
+                        <a href="{{ route('lang',['lang' => 'en']) }}" class="dropdown-item">
```
Comment (sonnguyen8x) [inline resources/views/layouts/master.blade.php]: conventions

**Comment by ghost** [comment]
ready

**Pull request merged**